lnd_services.go

```diff
@@ -324,6 +324,10 @@ func NewLndServices(cfg *LndServicesConfig) (*GrpcLndServices, error) {
 		log.Debugf("Wait for invoices to finish")
 		invoicesClient.WaitForFinished()
 
+		// TODO[carla] stop (w/ quit channel close) vs waitforfinished here?
```

Comment:
nit: TODO is not actually a TODO anymore? (Since afaict `stop()` does request stop and also waits.)

Reply:
ahyep, removing this. Renamed `stop` to `WaitForFinished` so that it's more consistent, I _think_ that's what I'm trying to say here as well.